api/src/main/scala/uk/ac/wellcome/platform/api/controllers/WorksController.scala

```diff
@@ -44,36 +43,19 @@ object IncludesValidation {
   def validateIncludes(includes: Option[String]): ValidationResult = {
     includes match {
       case Some(includesString) => {
-        val includesList = includesString.split(",").toList
-        val recognisedIncludes = getCaseClassFieldNames[WorksIncludes]
-        val badIncludes = includesList
-          .filterNot(recognisedIncludes.contains)
-        badIncludes.length match {
-          case 0 => Valid
-          case 1 => Invalid(s"includes: '${badIncludes.head}' is not a valid include")
-          case _ => Invalid(s"includes: ${badIncludes.mkString("'", "', '", "'")} are not valid includes")
+        WorksIncludes.apply(includesString) match {
+          case Left => Valid
```

Comment:
As a consequence of the above, you might end up wrapping the creation in a `Try` - alternatively if you see fit create a different (non-apply) factory method that returns an `Either`.